src/ethereum_test_tools/vm/opcode.py

```diff
@@ -205,171 +205,4898 @@ class Opcodes(Opcode, Enum):
+----
+- b: swapped value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- ignored value.
+- a: swapped value.
+
+Fork
+----
+Frontier
+
+Gas
+----
+3
+
+Source: [evm.codes/#9F](https://www.evm.codes/#9F)
+"""
+
+
+LOG0 = Opcode(0xA0, popped_stack_items=2)
+"""
+LOG3(offset, size) 
+----
+
+Description
+----
+Append log record with no topics
+
+Inputs
+----
+- offset: byte offset in the memory in bytes
+- size: byte size to copy
+
+Outputs
+----
+- None
+
+Fork
+----
+Frontier
+
+Gas
+----
+375 + 375 * topic_count + 8 * size + memory_expansion_cost
+
+Source: [evm.codes/#A0](https://www.evm.codes/#A0)
+"""
+
+LOG1 = Opcode(0xA1, popped_stack_items=3)
+"""
+LOG1(offset, size, topic1) 
+----
+
+Description
+----
+Append log record with one topics
+
+Inputs
+----
+- offset: byte offset in the memory in bytes
+- size: byte size to copy
+- topic1: 32-byte value
+
+Outputs
+----
+- None
+
+Fork
+----
+Frontier
+
+Gas
+----
+375 + 375 * topic_count + 8 * size + memory_expansion_cost
+
+Source: [evm.codes/#A1](https://www.evm.codes/#A1)
+"""
+
+LOG2 = Opcode(0xA2, popped_stack_items=4)
+"""
+LOG2(offset, size, topic1, topic2) 
+----
+
+Description
+----
+Append log record with two topics
+
+Inputs
+----
+- offset: byte offset in the memory in bytes
+- size: byte size to copy
+- topic1: 32-byte value
+- topic2: 32-byte value
+
+Outputs
+----
+- None
+
+Fork
+----
+Frontier
+
+Gas
+----
+375 + 375 * topic_count + 8 * size + memory_expansion_cost
+
+Source: [evm.codes/#A2](https://www.evm.codes/#A2)
+"""
+
+LOG3 = Opcode(0xA3, popped_stack_items=5)
+"""
+LOG3(offset, size, topic1, topic2, topic3) 
+----
+
+Description
+----
+Append log record with three topics
+
+Inputs
+----
+- offset: byte offset in the memory in bytes
+- size: byte size to copy
+- topic1: 32-byte value
+- topic2: 32-byte value
+- topic3: 32-byte value
+
+Outputs
+----
+- None
+
+Fork
+----
+Frontier
+
+Gas
+----
+375 + 375 * topic_count + 8 * size + memory_expansion_cost
+
+Source: [evm.codes/#A3](https://www.evm.codes/#A3)
+"""
+
+LOG4 = Opcode(0xA4, popped_stack_items=6)
+"""
+LOG4(offset, size, topic1, topic2, topic3, topic4) 
+----
+
+Description
+----
+Append log record with four topics
+
+Inputs
+----
+- offset: byte offset in the memory in bytes
+- size: byte size to copy
+- topic1: 32-byte value
+- topic2: 32-byte value
+- topic3: 32-byte value
+- topic4: 32-byte value
+
+Outputs
+----
+- None
+
+Fork
+----
+Frontier
+
+Gas
+----
+375 + 375 * topic_count + 8 * size + memory_expansion_cost
+
+Source: [evm.codes/#A4](https://www.evm.codes/#A4)
+"""
+
+
+RJUMP = Opcode(0xE0, data_portion_length=2)
+"""
+RJUMP(destination)
+----
+
+Description
+----
+Jump to a relative position in code.
+
+Inputs
+----
+- destination: The relative position to jump to.
+
+Fork
+----
+Revolution
```

Comment:
```suggestion
N/A
```
It's supposed to be included with EOF, which is aimed for the Prague hard-fork, but until it's formally considered for inclusion we can just put `N/A` here.